docs/source/user_guide/configuration_options.rst

```diff
@@ -0,0 +1,890 @@
+==============================
+Receptor Configuration Options
+==============================
+
+---------------------
+Receptor command line
+---------------------
+
+Command line arguments have the form: receptor [--<action> [<param>=<value> ...] ...]
+
+<action> are listed below.  When an action has parameters, they are listed in their own section.
+
+-------------
+Miscellaneous
+-------------
+
+.. list-table:: Miscellaneous
+    :header-rows: 1
+    :widths: auto
+
+    * - Action
+      - Description
+    * - --bash-completion
+      - Generate a completion script for the bash shell
+    * - --config <filename>
+      - Load additional config options from a YAML file
+    * - --control-service
+      - Run a control service
+    * - --help
+      - Show this help
+    * - --local-only
+      - Run a self-contained node with no backends
+    * - --log-level
+      - Set specific log level output
+    * - --node
+      - Node configuration of this instance (required)
+    * - --trace
+      - Enables packet tracing output
+    * - --version
+      - Show the Receptor version
```

Comment:
```suggestion
      - Displays the Receptor version.
```

Reply:
Aha! So it is "Receptor". This should change globally when referring to the project/product name.

Reply:
This came from the code. (help)  I also updated the text there.